apps/fortuna/src/keeper.rs

```diff
@@ -128,9 +128,80 @@ impl Default for KeeperMetrics {
 }
 
 impl KeeperMetrics {
-    pub async fn new(registry: Arc<RwLock<Registry>>) -> Self {
+    pub async fn new(
+        registry: Arc<RwLock<Registry>>,
+        chain_ids: Vec<String>,
+        provider_addresses: Vec<Address>,
```

Comment:
please take a vec of tuples as a single argument: `chain_ids_and_providers: Vec<(String, Address)>`